### PR TITLE
Opensource : 6.3 Everything about PIP

### LINKED ISSUE #2966: Opensource : 6.3 Everything about `PIP`
<hr>

## Description 📜

Opensource :   6.3 Everything about `PIP` 

### NOTE 👇
Please put your doc `6.Popular_Projects` with the name `6.3_PIP.md`

<hr>

### Domain

Open Source

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by GyaniRoman23**
<hr>

## Description 📜

<h2>FIxes #2966</h2>

<h3>Includes the following content:</h3> 

- Brief intro to Pip (imcluding references to official docs)

- A comprehensive refresher to python packages

- Python pip Tool
    - Installation
    - Using the requirements.txt File
    - Uninstalling Packages
    - Viewing all installed Packages
    - Viewing Details for an Installed Package
    - Freezing Package Versions

- pip vs pip3

-  Conclusion and some external references

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] OpenSource

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

### CODE REVIEW

**Review by mayankkuthar — COMMENTED** [review]
@GyaniRoman23 Good work! :octocat: 
### Suggestion:-

If you add 1-2 more images OR screenshots of running pip commands mentioned in the documentation it would be better. 😄

**Comment by GyaniRoman23** [comment]
> @GyaniRoman23 Good work! :octocat:
> 
> ### Suggestion:-
> If you add 1-2 more images OR screenshots of running pip commands mentioned in the documentation it would be better. 😄

Sure!! I am just waiting for mentor's reply. Once I get it, I'll add the suggested changes.

File: Open_Source/6.Popular_Projects/6.3_PIP.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+<p align="center">
+  <img src="https://files.realpython.com/media/What-is-PIP_Watermarked.c46f49dc33f9.jpg" />
```
Comment (Aryamanz29) [inline Open_Source/6.Popular_Projects/6.3_PIP.md:2]: Replace this image with another image or cropped that, coz there I see `Real Python logo` which we don't want in our `docs`

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

File: Open_Source/6.Popular_Projects/6.3_PIP.md (hunk 1 above)
Comment (GyaniRoman23) [reply Open_Source/6.Popular_Projects/6.3_PIP.md]: Hey !! Could you review it again. I have removed the watermark.

**Review by Aryamanz29 — APPROVED** [review]
LGTM
Thanks for contributing @GyaniRoman23  🎉🎉

**Pull request merged**